.expeditor/config.yml

```diff
@@ -23,6 +23,19 @@ merge_actions:
         - "Expeditor: Skip All"
       only_if: built_in:bump_version
 
+# Subscription to workload action of pull request merge to master branch on
+# omnibus-software project will allow us to trigger an uncached omnibus build
+# for omnibus-toolchain and angry-omnibus-toolchain when the omnibus-software
+# has modified software config files.
+subscriptions:
+  # chef/omnibus-software
+  - workload: pull_request_merged:chef/omnibus-software:master:*
+    actions:
+      - built_in:trigger_omnibus_expcache_build:
+          only_if_modified:
+      	    - config/software/*
+            - config/patches/*
```

Comment:
[nit] you want to line up the hyphens since it's a list

Reply:
Its funny my editors show them in line. Will force push again.